### DISCUSSION TITLE
[1406850]廖添丁：绝代凶贼之末日

### DISCUSSION THREAD

**Discussion by Psicxto** [pjy612/SteamManifestCache#159] [Share]
`appid:1406850`

```
1406851;8a1f9d6033a9c58f1b2e131bd984f57d15b48162ec6d0f6c6df575a6403aea7c
```

邮箱:vrednow@outlook.com
code: C3FB749C7F4638F1

**Comment by pjy612** [comment]
https://steamdb.info/app/1406850/depots/
缺中文语言包 1406852 的 密钥吧

**Reply by Psicxto** [reply]
我又搜了一遍config.vdf没有这个1406852，但是游戏运行时就直接有简体中文啊。。奇怪

**Reply by pjy612** [reply]
行，那先这样吧。
查收
